snapshots/sample/app/src/androidTest/kotlin/com/emergetools/snapshots/sample/MediaPickerTest.kt

```diff
@@ -0,0 +1,35 @@
+package com.emergetools.snapshots.sample
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.test.junit4.ComposeContentTestRule
+import androidx.compose.ui.test.junit4.createComposeRule
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.emergetools.snapshots.EmergeSnapshots
+import com.emergetools.snapshots.compose.SnapshotVariantProvider
+import com.emergetools.snapshots.sample.ui.MediaPickerDemo
+import com.emergetools.snapshots.shared.ComposePreviewSnapshotConfig
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+public class MediaPickerTest {
```

Comment:
Copied over and tweaked one of our generated classes, made it really easy to reproduce and get the stacktrace.

Reply:
Might be good to add a comment here about the issue and why it's our only integration test 😂